index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <h1>🍔 도 와 줘 햄 식 ! 🍟</h1>
+    </header>
+    <div class="container">
+      <aside class="sideBar">
+        <div class="other-menu">
+          <h2 class="other-menu-title">다른거 먹자🍔</h2>
+          <div class="other-menu-macdonald">맥도날드</div>
+          <div class="other-menu-burgerking">버거킹</div>
+        </div>
+      </aside>
+      <section class="burger">
+        <article class="burger-card">
+          <img src="https://www.mcdelivery.co.kr/kr//static/1648609546423/assets/82/products/4940.png?" alt="리코타치킨 버거">
+          <div class="burger-detail">
+            <span class="burger-detail-name">리코타 치킨 버거</span>
+            <span class="burger-detail-price">₩7,300</span>
+            <span class="burger-detail-ment">얼마나 맛있게요?</span>
```

Comment:
```suggestion
            <h3 class="burger-detail-name">리코타 치킨 버거</h3>
            <p class="burger-detail-price">₩7,300</p>
            <p class="burger-detail-ment">얼마나 맛있게요?</p>
```
span테그를 지양하고, 시맨틱한 테그를 쓰는 것이 좋을 것 같아요!
article테그에는 하나 이상의 heading테그가 권장돼서, 가장 중요한 내용을 h테그로 써주고, 그 외 정보들을 P테그를 사용하면 좋을 것 같습니다!